### DISCUSSION TITLE
Will Prism.Templates have an update?

### DISCUSSION THREAD

**Discussion by sact1909** [PrismLibrary/Prism#3090] [General]
I installed Prism.Templates pack from the day it came out I saw it still has the same version 8.1.97 and after I create a new project I have to update the library and fix some things on the code, that is not a big problem for me, but I would like to know if this will have an update in the future could be awesome and it is a helpful template pack.

**Comment by dansiegel** [comment]
@sact1909 the Templates for Prism 9.0 are currently only available for those with a Commercial Plus license.
